src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Square extends Figure implements GetFigureArea {
+    private double side;
+
+    @Override
+    public String toString() {
+        return "Figure - Square: side " + this.side
+                + " ,area " + this.getArea() + " color "
+                + this.color;

```

Comment:
The `color` field is accessed directly here. Consider using a getter method if encapsulation is a concern, especially if the `color` field is private in the `Figure` class.